### ISSUE TITLE
readme.md needs updated

### ISSUE THREAD

**Issue by wekempf** [filipw/dotnet-script#101]
With the latest version, the readme.md is outdated. Need to talk about how to "install" via the zip package (I'd like to see package manager support, such as Chocolatey on Windows and apt-get or other managers on other platforms). Also need to remove the project.json stuff and show the code using the new #r directives.

```csharp
#r "nuget:AutoMapper,5.1.1"
#r "nuget:Newtonsoft.Json,9.0.1"

using Newtonsoft.Json;
using AutoMapper;

Console.WriteLine("hello!");

var test = new { hi = "i'm json!" };
Console.WriteLine(JsonConvert.SerializeObject(test));

Console.WriteLine(typeof(MapperConfiguration));
```

Just some feedback... this is a huge step forward, and now it's truly a scripting language. The only downside here is in the performance. The startup time when trying to run the above code is quite significant. Still, this is an awesome way to experiment with C# code.

**Comment by filipw** [comment]
you are absolutely right, and I'm really sorry - it's on my TODO 📚
We are also working on a better installation story 👍

**Comment by seesharper** [comment]
Want me to take a look at the readme?

**Comment by seesharper** [comment]
#126

**Comment by seesharper** [comment]
Fixed by #126
